docs/insights/README.md

```diff
@@ -104,5 +104,18 @@ For example, to increase the line length limits:
     ]
 ```
 
-You can also configure the `exclude` parameter on each insight, to disallow a
+You can also configure the `exclude` parameter on each insight, to disallow an
 insight on a specific file.
+
+For example, to remove "Unused Parameters" Insight only for some file:
```

Comment:
It can be reworded if you want. 

I initially wanted to put "only for some Class that implement interface without using all parameters" but i found it "so" specific.